Framework/Algorithms/src/SumSpectra.cpp

```diff
@@ -95,7 +129,7 @@ void SumSpectra::exec() {
   this->m_yLength = static_cast<int>(localworkspace->blocksize());
 
   // Check 'StartSpectrum' is in range 0-m_numberOfSpectra
-  if (m_minWsInd > m_numberOfSpectra) {
+  if (m_minWsInd >= m_numberOfSpectra) {
```

Comment:
How about testing *StartWorkspaceIndex* in `validateInputs` as well? You would need to ensure that `m_minWsInd < m_maxWsInd` and `m_minWsInd < m_numberOfSpectra`.